lib/zone-spec/fake-async-test.ts

```diff
@@ -297,8 +297,15 @@
               break;
             case 'XMLHttpRequest.send':
               throw new Error('Cannot make XHRs from within a fake async test.');
+            case 'requestAnimationFrame':
+            case 'webkitRequestAnimationFrame':
+            case 'mozRequestAnimationFrame':
+              // Simulate a requestAnimationFrame by using a setTimeout with 16 ms.
+              // (60 frames per second)
+              task.data['handleId'] = this._setTimeout(task.invoke, 16, (task.data as any)['args']);
+              break;
             default:
-              task = delegate.scheduleTask(target, task);
+              throw new Error('Unknown macroTask scheduled in fake async test: ' + task.source);
```

Comment:
I'd say that we should consider whether it's meaningful to make them available - but in the meantime, throwing an error is much more useful and clearer than silently passing through.